### PR TITLE
Add list item icon action

### PR DESCRIPTION
**Pull request by kelianClerc**
Capture d’écran 2022-12-08 à 10 00 35

### CODE REVIEW

File: library-compose/src/main/java/com/spendesk/grapes/compose/icons/GrapesSurface.kt [hunk 1]
```diff
@@ -31,9 +33,15 @@ fun GrapesSurface(
         GrapesConfigurationStatus.BLOCKED -> Pair(GrapesTheme.colors.mainPrimaryLightest, GrapesTheme.colors.mainPrimaryLighter)
     }
 
+    val border = if (hasBorder) {
+        BorderStroke(1.dp, borderColor)
+    } else {
+        null
+    }
+
```
Comment (jpbarbaud) [inline library-compose/src/main/java/com/spendesk/grapes/compose/icons/GrapesSurface.kt:41]: Maybe you can replace by 
```
val border = BorderStroke(…).takeIf { hasBorder }
```

File: library-compose/src/main/java/com/spendesk/grapes/compose/icons/StatusInformationIcon.kt [hunk 2]
```diff
@@ -14,42 +20,56 @@ import com.spendesk.grapes.compose.theme.GrapesTheme
  * @since : 19/05/2022, Thu
  **/
 
-enum class Size {
-    S,
-    M,
-    L
+enum class Size(val surfaceSize: Dp, val iconSize: Dp) {
```
Comment (jpbarbaud) [inline library-compose/src/main/java/com/spendesk/grapes/compose/icons/StatusInformationIcon.kt:23]: What is the difference between `Dp` and `dp` ?

File: library-compose/src/main/java/com/spendesk/grapes/compose/listitem/ListItem.kt [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@Composable
+fun ListItem(
+    configuration: ListItemConfiguration,
+    modifier: Modifier = Modifier
+) {
+    when (configuration) {
+        is ListItemConfiguration.IconAction -> IconAction(iconActionConfiguration = configuration, modifier)
+    }
```
Comment (jpbarbaud) [inline library-compose/src/main/java/com/spendesk/grapes/compose/listitem/ListItem.kt:37]: a `when` with single entry ?

File: library-compose/src/main/java/com/spendesk/grapes/compose/theme/GrapesTheme.kt [hunk 4]
```diff
@@ -34,11 +40,30 @@ fun GrapesTheme(
     darkTheme: Boolean = isSystemInDarkTheme(),
     content: @Composable () -> Unit
 ) {
+    val rippleIndication = rememberRipple()
```
Comment (jpbarbaud) [inline library-compose/src/main/java/com/spendesk/grapes/compose/theme/GrapesTheme.kt:43]: So easy to create a ripple 😮

File: library-compose/src/main/java/com/spendesk/grapes/compose/icons/StatusInformationIcon.kt (hunk 2 above)
Comment (kelianClerc) [reply library-compose/src/main/java/com/spendesk/grapes/compose/icons/StatusInformationIcon.kt:23]: Dp is the type et .dp is the extension fonction Float.dp(): Dp to convert a float into a Dp

File: library-compose/src/main/java/com/spendesk/grapes/compose/listitem/ListItem.kt (hunk 3 above)
Comment (kelianClerc) [reply library-compose/src/main/java/com/spendesk/grapes/compose/listitem/ListItem.kt:37]: This item will later have many entries according to the design system. I set it up like that for the future. Do you prefer removing it for now and adding it only when needed?

Comment (jpbarbaud) [reply library-compose/src/main/java/com/spendesk/grapes/compose/listitem/ListItem.kt:37]: Nope it's ok for me if they will have more entries later

**Review by jpbarbaud — APPROVED** [review]

**Pull request merged**